### PR TITLE
docs(README): fixed broken links, typos, and grammatical errors

### LINKED ISSUE #1840: Broken Links in README.md
### Description

- The link of [general documentation](http://www.fossology.org/projects/fossology/wiki/User_Documentation) in Readme.md is broken
- In the about section of Readme.md the link of fossology website is given as http://fossology.org/ which should be changed to https://fossology.org/ (changing http to https )

#### How to reproduce
Visit the [general documentation](http://www.fossology.org/projects/fossology/wiki/User_Documentation) link in Readme.md

#### Screenshots

image

### LINKED ISSUE #1878: Docs: Update Readme.md
### Description
A proposal to add/update the followings points in the [README.md](https://github.com/fossology/fossology/blob/master/README.md) file
1. Add the link of fossology [YouTube channel](https://www.youtube.com/channel/UCZGPJnQZVnEPQWxOuNamLpw) (as a badge at the top of readme)
2. In the About section the link of [Check out Who Uses FOSSology!](http://www.fossology.org/projects/fossology/wiki/WhoUsesFOSSology) is broken 
and can be changed to [https://www.fossology.org/#footer-outer](https://www.fossology.org/#footer-outer)
3. In the Installation section the wiki link of [Install from Source](https://github.com/fossology/fossology/wiki/Install-from-Source) can be added
4. In the Documentation section the link of [general documentation](http://www.fossology.org/projects/fossology/wiki/User_Documentation) is broken 
so it can be changed temporarily to https://www.fossology.org/get-started/basic-workflow/ or https://wiki.fossology.org/start or https://fossology.github.io/
5. Adding a Test Instance
The website https://wiki.fossology.org/start states that a demo server is present at http://83.169.21.23/fossology/ but it is presently not working.
Another website https://www.fossology.org/get-started/ contains the link of a test instance present at https://fossology.osuosl.org/  which is a working version of fossology [Version 3.7.0]
It would be great if we can add the latter link in readme as it may be helpful for new users to learn using fossology without going through the hassle of installation process.

### Versions

* Last commit id on master: `19d85bd`

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Description

Updated Readme.md

### Changes

- Added link of fossology YouTube channel 
- Fixed broken links
- Added link of a test instance
- Fixed typos
- Fixed grammatical errors

Closes #1878 
Closes #1840

### CODE REVIEW

**Review by shaheemazmalmmd — APPROVED** [review]
Code looks good.

**Comment by Aman-Codes** [comment]
Also added link of the test instance. Kindly review

File: README.md [hunk 1]
```diff
@@ -69,18 +71,29 @@ cd fossology/
 vagrant up
 ```
 
-The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/) and user can login the credentials using
+The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/) and the user can log in using the credentials
```
Comment (GMishx) [inline README.md:74]: The word `login` is correct in this context IMO.

Comment (Aman-Codes) [reply README.md:74]: It was suggested by Grammarly (ref: https://grammarist.com/spelling/log-in-login/ )
Does the below sentence sounds good?
```
The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/). The login credentials are
```

Comment (GMishx) [reply README.md:74]: Ah ha, I learned something new today. Thanks for sharing the link.
Then grammatically both sentences are correct :+1:

**Pull request merged**